odk1-src/form-logic.rst

```diff
@@ -240,44 +264,71 @@ without answering the question.
-  is taken literally as the default value.
-  Quotes should **not** be used to wrap string values,
-  unless you actually want those quote marks to appear
-  in the default response value.
+The text in the :th:`default` column for a question is taken literally as the default value. Quotes should **not** be used to wrap values, unless you actually want those quote marks to appear in the default response value.
 
-.. rubric:: XLSForm
+In the example below, the "Phone call" option with underlying value ``phone_call`` will be selected when the question is first displayed. The enumerator can either keep that selection or change it.
+
+.. rubric:: XLSForm to select "Phone call" as the default contact method
 
 .. csv-table:: survey
   :header: type, name, label, default
-  
+
   select_one contacts, contact_method, How should we contact you?, phone_call
-  
+
 .. csv-table:: choices
   :header: list_name, name, label
-  
+
   contacts, phone_call, Phone call
   contacts, text_message, Text message
   contacts, email, Email
+
+.. _dynamic-defaults:
+
+Dynamic defaults
+----------------
+
+.. warning::
   
+  Support for :ref:`dynamic defaults <dynamic-defaults>` was added in Collect v1.24.0. Form conversion requires XLSForm Online ≥ v2.0.0 or pyxform ≥ v1.0.0.
+
+If you put an expression in the :th:`default` column for a question, that expression will be evaluated once when a record is first created from a blank form definition. This allows you to use values from outside the form like the current date or the :ref:`server username <metadata>`.
+
+.. rubric:: XLSForm to set the current date as default
+
+.. csv-table:: survey
+  :header: type, name, label, default
+
+  date, fever_onset, When did the fever start?, now()
+
+In the example below, if a username is set either in the :ref:`server configuration <server-settings>` or the :ref:`metadata settings <form-metadata-settings>`, that username will be used as the default for the question asked to the enumerator.
+
+.. rubric:: XLSForm to set the default username as the server username
+
+.. csv-table:: survey
+  :header: type, name, label, default
+
+  username, username
+  text, confirmed_username, What is your username?, ${username}
+
+If enumerators are creating records representing groups of things with the same property, dynamic defaults can be combined with :ref:`last saved <last-saved>`.
+
+Dynamic defaults in repeats are evaluated when a new repeat instance is added.
+
 .. tip:: 
-  :name: dynamic-defaults
+  :name: defaults-from-form-data
 
-  You may want to use a previously entered value as a default,
-  but the :th:`default` column does not accept dynamic values.
+  You may want to use a value filled out by the enumerator as a default for another question that the enumerator will later fill in. Dynamic defaults can't be used for this because they are evaluated once when a record is first created which is before an enumerator fills in any data.
```

Comment:
It'd be nice to say some of this upfront. "Dynamic defaults cannot be used to pre-populate answers to questions that an enumerator has not yet seen. This is because defaults are evaluated once when a record is first created"